<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Admin" failures="1" tests="6" name="Default test(failed)" time="0.113" errors="0" timestamp="2024-07-05T13:20:22 IST">
  <testcase classname="practice.InvoiceListenerRetryTest" name="activateSim" time="0.014">
    <skipped/>
  </testcase> <!-- activateSim -->
  <testcase classname="practice.InvoiceListenerRetryTest" name="activateSim" time="0.004">
    <skipped/>
  </testcase> <!-- activateSim -->
  <testcase classname="practice.InvoiceListenerRetryTest" name="activateSim" time="0.002">
    <skipped/>
  </testcase> <!-- activateSim -->
  <testcase classname="practice.InvoiceListenerRetryTest" name="activateSim" time="0.004">
    <skipped/>
  </testcase> <!-- activateSim -->
  <testcase classname="practice.InvoiceListenerRetryTest" name="activateSim" time="0.002">
    <skipped/>
  </testcase> <!-- activateSim -->
  <testcase classname="practice.InvoiceListenerRetryTest" name="activateSim" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [Login] but found []">
      <![CDATA[java.lang.AssertionError: expected [Login] but found []
at practice.InvoiceListenerRetryTest.activateSim(InvoiceListenerRetryTest.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- activateSim -->
</testsuite> <!-- Default test(failed) -->
